docs/accessibility-resources.md

```diff
@@ -1,42 +1,58 @@
 # Accessibility Resources 
 
-Accessibility is an ongoing journey, and should be just part of how we design and build software. The B.C. Government is working towards WCAG 2.2 AA compliance for all internal and external web content. 
+Ensuring accessibility is an ongoing process integral to software design and development. The B.C. Government is committed to achieving WCAG 2.2 AA compliance for both internal and external web content.
 
-These resources may be useful to build or build upon your accessibility practice. Please be mindful of protecting PII while using any web tools. 
+Below are resources to support and enhance your accessibility initiatives, including privacy information where available. Please exercise caution to safeguard personally identifiable information (PII) when utilizing web tools.
  
 ## Learn about accessibility
-A11y Project checklist
-DigitalA11y's definitive guide
-W3C's WCAG-EM Report Tool 
-Web Content Accessibility Guidelines (WCAG) 2.2
-A11y Project
-WebAim's Quick Reference list
-Microsoft Inclusive Design
+- [A11y Project checklist](http://www.a11yproject.com/checklist)
+- [A11y Project](http://www.a11yproject.com/)
+- [DigitalA11y's definitive guide](https://www.digitala11y.com/a-definitive-guide-on-how-to-perform-a-web-accessibility-audit/)
+- [W3C's WCAG-EM Report Tool](https://www.w3.org/WAI/eval/report-tool/evaluation/define-scope)
+- [Web Content Accessibility Guidelines (WCAG) 2.2](https://www.w3.org/TR/WCAG22/)
+- [WebAim's Quick Reference list](https://webaim.org/resources/quickref/)
+- [Microsoft Inclusive Design](https://inclusive.microsoft.design/)
+<hr/>
```

Comment:
no hrs please - if we need to tweak our styles to make the spacing better here for better page hierarchy I am happy to do that in a separate ticket

Reply:
Removed.